langkit/templates/pkg_implementation_spec_ada.mako

```diff
@@ -52,6 +52,26 @@ private package ${ada_lib_name}.Implementation is
    type Analysis_Context_Type;
    type Internal_Context is access all Analysis_Context_Type;
 
+   Unexpected_Call_Depth : exception;
```

Comment:
This is not used

Reply:
Nice catch: I’ve pushed an update to use it. Thanks!